astropy/coordinates/representation/geodetic.py

```diff
@@ -94,10 +96,93 @@ def from_cartesian(cls, cart):
         Converts 3D rectangular cartesian coordinates (assumed geocentric) to
         geodetic coordinates.
         """
+        # Compute geodetic/planetodetic angles
         lon, lat, height = erfa.gc2gde(
             cls._equatorial_radius, cls._flattening, cart.get_xyz(xyz_axis=-1)
         )
-        return cls(lon, lat, height, copy=False)
+        return cls(
+            lon,
+            lat,
+            height,
+            copy=False,
+        )
+
+
+@format_doc(geodetic_base_doc)
+class BaseBodycentricRepresentation(BaseRepresentation):
+    """Representation of points in bodycentric 3D coordinates.
+
+    Subclasses need to set attributes ``_equatorial_radius`` and ``_flattening``
+    to quantities holding correct values (with units of length and dimensionless,
+    respectively).
+    Longitudes are east positive and span from 0 to 360 degrees by default.
```

Comment:
Again, I'd remove this for now and instead describe how the parameters are used (and how it is different from geodetic).

Alternatively, do not use `geodetic_base_doc`, but instead write a complete docstring here that also describes how `lat` is defined.